src/style.css

```diff
@@ -0,0 +1,138 @@
+body,
+p {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--4 > .stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 > .stars__star:nth-of-type(n + 2) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  display: flex;
+}
+
+.container {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+}
+
+.container > img {
+  width: 160px;
+  height: 134px;
+  padding: 16px 3px 40px;
+}
+
+.quantity {
+  font-size: 10px;
+  height: 16px;
+  line-height: 18px;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.price-name {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: rgba(97, 96, 112, 1);
+}
+
+.price-value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.container > a {
+  box-sizing: border-box;
+  text-transform: uppercase;
+  background-color: rgba(0, 172, 220, 1);
+  text-decoration: none;
+  border-radius: 5px;
+  height: 40px;
+  line-height: 40px;
+  text-align: center;
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 4px 0;
```

Comment:
use 1 direction approach for paddings 
it will be more readable if you add only top or bottom paddings